pkg/pillar/types/downloadertypes.go

```diff
@@ -224,3 +228,20 @@ type DatastoreContext struct {
 	Password        string
 	Region          string
 }
+
+// DownloadedParts keeps information about downloaded parts of blob
+type DownloadedParts struct {
+	PartSize int64           // not used now, but may be used when partsize will be variable
```

Comment:
What's the intended meaning? Size of one partition? Total size? 
Does it make sense introducing it if its use isn't well-defined and commented?

Reply:
I put here size of part (maximum size). It may be useful if we will decide to change part size in the future (and will do some kind of re-calculation what we need to download with new part sizes).

Reply:
OK so then the comment should be modified since it is used in the sense that it it set.
Is this a correct description?
// Set to the maximum partition size

Reply:
Modified, thanks for the comment